### ISSUE TITLE
Support other SDR/RSP devices (like HackRF One and/or SDRplay SRPdx)

### ISSUE THREAD

**Issue by ralphbolliger** [moneriomaa/openear#49]
At the moment I own a **HackRF One** and a **SDRplay RSPdx**. Now I dicovered OpenEar and would like to play with it. But unfortunately I'm not able to di this because OpenEar doesn't seem to support both devices I have now.

**Is there any plans to support HackRF One and/or SDRplay SRPdx?**

Or will it be possible to support external audio input via an _Virtual Audio Cable_?

**Comment by moneriomaa** [comment]
about other hardware supporting it under development
 _**"input via an Virtual Audio Cable? "**_ it added on version 1.7.0

**Comment by Bangs69** [comment]
I'm really looking forward to hackrf compatibility, this is already great with an RTL-SDR. Thank you for continuing to develop the project!

**Comment by moneriomaa** [comment]
you're welcome
thanks for your attention
about other hardware support, i try to add it on next releases
but before this, user can use audio input mode for decoding

**Comment by Bangs69** [comment]
Thank you!
Do you have any plans to release a Raspberry Pi version? This already has more functionality than the few main hackrf portapack h1/h2 firmware choices. 
If I had a little Pi with touch LCD and a hackrf runnig Openear, I would happily toss my portapack h2 in a heartbeat!
I imagine most of the other owners of these devices would love to have another powerful and portable option available too.
Would this be possible?

**Comment by moneriomaa** [comment]
you're welcome
about Raspberry Pi version, good idea and i think it is possible (some Raspberry Pi processing power must be check)
but currently it is not in plan, but interesting field for OpenEar future.

**Comment by Bangs69** [comment]
Definitely! A lot of people who bought the portapack h2 (myself included) are having a lot of issues with them, the charge circuit is very noisy and in some cases (again... myself included) the battery capacity has been upgraded to 3500mah and it needs to be rotated 90 degrees to fit. The problem here is that the noisy charge circuit is directly above the amplifier chip with no shielding, this makes the rf noise even worse and the whole thing won't power up if you remove the battery 😑
A cheap option like the Raspberry Pi will be something that more and more people are looking for in the coming months/years as they give up on their portapacks. You could even purchase a bulk order of them and sell them with the software preloaded, it could be a really good way to inject funds into your project!
Sign me up for the first 2 units!

**Comment by moneriomaa** [comment]
thanks for sharing your experience

**Comment by IgrikXD** [comment]
Good evening! It would be cool if, in addition to the above devices, support for Airspy receivers appeared in future releases.
